webviz_ert/plugins/_webviz_ert.py

```diff
@@ -30,7 +30,7 @@ def init_state(project_root: pathlib.Path) -> MutableMapping[str, Any]:
             if WebvizErtPluginABC._state_path is None:
                 WebvizErtPluginABC._state_path = (
                     project_root
-                    / f"storage/webviz/{WebvizErtPluginABC._state_file_name}"
+                    / f".webviz/{WebvizErtPluginABC._state_file_name}"
```

Comment:
what is the working directory of webviz-ert? I guess that is where this will end up. Is it the config directory, or where the user ran the ert command from? Or undefined?

Reply:
The working directory of webviz-ert is some temp folder. 

This will end up in the folder where the user ran the command from, probably this should change to the path of the ert config file.